slack_bolt/request/internals.py

```diff
@@ -226,6 +226,14 @@ def extract_function_bot_access_token(payload: Dict[str, Any]) -> Optional[str]:
     return None
 
 
+def extract_function_inputs(payload: Dict[str, Any]) -> Optional[Dict[str, Any]]:
```

Comment:
Just want to verify that there's no use cases we can think of (specifically interactivity handling, given how many elements are supported) that will also have `inputs` off of the `event`. Basically, that we're sure that `event.inputs` will always be function inputs and not anything else.

Reply:
Good point 💯 I'm not sure, and not sure how we could validate this but I'll change this so that is it more resilient to other formats

Reply:
Actually, this might be unclear in the code but bolt will only try to extract_function_inputs if the there is a [function execution id found int he payload](https://github.com/WilliamBergamin/bolt-python/pull/8/files#diff-f51f0eb4ac24e7458dd2a2674e905634166f253d42b875085ead571d2c0ced8dR48)

Since the inputs will only be extracted if the payload has other function event related fields I think we can safely assume that `inputs` will always be function inputs and not anything else

@seratch let us know what you think of this assumption

Reply:
Good call @misscoded! 

[Here is the code in the deno SDK that handles this situation](https://github.com/slackapi/deno-slack-runtime/blob/main/src/dispatch-payload.ts#L144-L145).

I believe Will's code here in bolt mirrors the deno SDK logic. The only difference is the final fallback: in the deno SDK, we return an empty object. Here, we return `None`. I will leave it up to y'all to decide if we should align the two, or if there's a specific Python convention that should be followed.